test-apps/presentation-test-app/src/backend/electron/ElectronMain.ts

```diff
@@ -19,7 +21,14 @@ export default async function initialize(rpcInterfaces: RpcInterfaceDefinition[]
     developmentServer: process.env.NODE_ENV === "development",
     ipcHandlers: [SampleIpcHandler],
   };
-  await ElectronHost.startup({ electronHost });
+  const iModelHost = new IModelHostConfiguration();
+  iModelHost.authorizationClient = new ElectronAuthorizationBackend({
+    clientId: "imodeljs-electron-test",
+    redirectUri: "http://localhost:3000/signin-callback",
+    scope: "openid email profile organization itwinjs",
+  });
+  await ElectronHost.startup({ electronHost, iModelHost });
+  await (IModelHost.authorizationClient as ElectronAuthorizationBackend).initialize();
```

Comment:
I wouldn't cast it like this but instead call the `initialize` directly on the client.

``ts
const authClient = new ElectronAuthorizationBackend({
    clientId: "imodeljs-electron-test",
    redirectUri: "http://localhost:3000/signin-callback",
    scope: "openid email profile organization itwinjs",
  });
  await ElectronHost.startup({ electronHost, iModelHost });
  await authClient.initialize();
```

Reply:
What's the reasoning to require the initialize after the ElectronHost startup? Is it due to the creation of the Ipc channels?

Reply:
Its been a few weeks since doing this but I can't think of a reason why it would matter. The IPC handlers are setup when the client is created, not when it's initialized. I can change them all to initializing before the `Electron.Host.startup(...)` if you want - then I'll double check whenever the electron package is published to NPM and I can test it again.